src/views/TabView/TabBarBottom.js

```diff
@@ -71,10 +71,18 @@ export default class TabBarBottom extends PureComponent<DefaultProps, Props, voi
       inputRange,
       outputRange,
     });
+    const label = this.props.getLabelText(scene);
+    if (typeof label === 'string') {
+      return (
+        <Animated.Text style={[styles.label, { color }]}>
+          {label}
+        </Animated.Text>
+      );
+    }
     return (
```

Comment:
I think this can just be `return label`. `labelStyle` is supposed to be text style